lib/cfn-nag/util/enforce_secrets_manager.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require 'cfn-nag/util/truthy.rb'
+
+# Migrated from multiple classes, with some modifications
+# Returns true if the provided key_to_check is a no-echo parameter
+# without a default value; false otherwise.
+def secrets_manager_property_value?(_cfn_model, key_to_check)
+  if key_to_check.is_a? String
+    if key_to_check.start_with? '{{resolve:secretsmanager:'
+      if key_to_check.end_with? '}}'
+        return true
+      end
+    else
+      return false
+    end
+  end
+  false
+end
```

Comment:
I am surprised rubocop is OK with these nested ifs. I would assume it would prefer something like:

```
def secrets_manager_property_value?(_cfn_model, key_to_check)
  return false unless key_to_check.is_a? String
  return false unless key_to_check.start_with? '{{resolve:secretsmanager:'
  return false unless key_to_check.end_with? '}}'

  true
end
```

I don't feel super strongly either way